src/Components/Booth/Regist/Location/RegistLocationPage.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+
+interface EventBookingProps {
+  imageSrc: string;
+  rows: string[];
+  seatsPerRow: number[];
+  bookingStatus: {
+    confirmed: string[];
+    reserved: string[];
+    available: string[];
+  };
+}
+
+const EventBooking: React.FC<EventBookingProps> = ({
+  imageSrc,
+  rows,
+  seatsPerRow,
+  bookingStatus,
+}) => {
+  const [selectedSeats, setSelectedSeats] = useState<string[]>([]);
+
+  const getColorClass = (status: string) => {
+    switch (status) {
+      case "available":
+        return "bg-yellow-400";
+      case "reserved":
+        return "bg-red-400";
+      case "confirmed":
+        return "bg-gray-400";
+      default:
+        return "";
+    }
+  };
```

Comment:
```suggestion
 const colorClass = {
   available: "bg-yellow-400",
   reserved: "bg-red-400",
   confirmed: "bg-gray-400"
 }
```
취향 차이겠지만 switch문 대신 객체를 사용하는 방법도 있을 수 있겠습니다.
다만 해당 방식을 사용했을 때 default처리를 하지 못해 key값을 못 찾으면 undefined를 return하게 될 것 같습니다.

아래와 같이 사용됨
```typescript
colorClass["available1"]
```

Reply:
오 훨씬 깔끔하게 보이네요 사실 좌석의 상태 데이터는 무조건 넘어오게 되어있으므로 아래 방법을 사용하는 것이 여러모로 좋아보입니다. 수정해보겠습니다!